sql/core/src/main/scala/org/apache/spark/sql/expressions/Window.scala

```diff
@@ -113,12 +113,12 @@ object Window {
    * Creates a [[WindowSpec]] with the frame boundaries defined,
    * from `start` (inclusive) to `end` (inclusive).
    *
-   * Both `start` and `end` are relative positions from the current row. For example, "0" means
+   * Both `start` and `end` are relative positions to the current row. For example, "0" means
```

Comment:
I'll fix it to be more accurate (that's the purpose of this particular change so the more accurate the merrier). Thanks!